app/controllers/regulatory-statements/edit.js

```diff
@@ -183,6 +186,12 @@ export default class RegulatoryStatementsRoute extends Controller {
     ];
   }
 
+  get activeNode() {
+    if (this.controller) {
+      return getActiveEditableNode(this.controller.activeEditorState);
```

Comment:
`getActiveEditableNode()` seems to get `undefined` whenever an ember-node is selected, which then causes an exception in the snippet plugin, leaving the editor in a partially-working state. For example, by adding/clicking on a link node.

Reply:
getting the same error, will check if I can reproduce it in the plugins repo.

Reply:
This should be fixed now. A conditional checking if `activeNode` is defined was missing.